ocs_ci/ocs/ui/views.py

```diff
@@ -1029,6 +1029,14 @@
         "//li[normalize-space()='StorageSystem details']",
         By.XPATH,
     ),
+    "storagesystem-details-compress-state": (
+        "#compressionStatus",
+        By.CSS_SELECTOR,
+    ),
+    "storagecluster-blockpool-details-compress-status": (
+        "article[data-test-id='compression-details-card'] dd[class='co-overview-details-card__item-value']",
```

Comment:
What is an article locator?

Reply:
it is a CSS linking article - compression-details card with its trailing value